nav2_docking/opennav_docking/include/opennav_docking/utils.hpp

```diff
@@ -183,6 +185,17 @@ inline double l2Norm(const geometry_msgs::msg::Pose & a, const geometry_msgs::ms
     delta_angle * delta_angle);
 }
 
+inline DockDirection getDockDirectionFromString(const std::string & direction)
```

Comment:
Why not just have this be a boolean so there doesn't need to be a parser?

Reply:
Yes, with a ```bool``` it would be much simpler, but I thought that since we are going to modify it, we can leave the door open for future community plugins. For example, a robot that has omnidirectional wheels and wants to dock laterally.

Reply:
Eh ok. I typed out a whole thing but we can go with this :-)